sippy-ng/src/component_readiness/ComponentReadiness.js

```diff
@@ -401,6 +404,11 @@ export default function ComponentReadiness(props) {
   const [isLoaded, setIsLoaded] = React.useState(false)
   const [data, setData] = React.useState({})
 
+  const [regressedTestDialog, setRegressedTestDialog] = React.useState(false)
```

Comment:
Suggest making this a query parameter so the modal is open when you hit the back button (or share a link)

```suggestion
  const [regressedTestDialog = false, setRegressedTestDialog] = useQueryParam(
    'regressedModal',
    BooleanParam
  )```

In order to make this work nicely you'll have to yoink this parameter out of the URL in the `key` since it doesn't require a new API call

Example here:

https://github.com/openshift/sippy/commit/d7291880f92d29c9fb72725cb3c5add6b00f019d#diff-0dd37ad68b4331fc273d39705d19e402ed1eba635da3b349ff622c8dd09f3c3eR486